docs/landing/data/releases.toml

```diff
@@ -16,78 +21,11 @@ current = "3.12.0"
   docs = "./3.10"
   api = "./3.10/javadoc"
 
-[[versions]]
-  version = "3.9.0"
-  docs = "./3.9"
-  api = "./3.9/javadoc"
-
-[[versions]]
-  version = "3.8.2"
-  docs = "./3.8"
-  api = "./3.8/javadoc"
-
-[[versions]]
-  version = "3.7.1"
-  docs = "./3.7"
-  api = "./3.7/javadoc"
-
-[[versions]]
-  version = "3.6.4"
-  docs = "./3.6"
-  api = "./3.6/javadoc"
-
-[[versions]]
-  version = "3.5.0"
-  docs = "./3.5"
-  api = "./3.5/javadoc"
-
-[[versions]]
-  version = "3.4.3"
```

Comment:
Let's at least go back to here, since MongoDB 3.4 was only recently EOLed

Reply:
Done